prelude/python_bootstrap/tools/BUCK.v2

```diff
@@ -6,9 +6,16 @@ source_listing()
 
 prelude = native
 
+prelude.export_file(
+    name = "win_python_wrapper.bat",
+    src = "win_python_wrapper.bat",
+    target_compatible_with = ["config//os:windows"],
+    visibility = ["PUBLIC"],
+)
+
 prelude.sh_binary(
     name = "win_python_wrapper",
-    main = "win_python_wrapper.bat",
+    main = ":win_python_wrapper.bat",
```

Comment:
I don't see why this is needed.  An `attrs.source()` should show up in the input graph, so this layer of indirection doesn't seem to do anything.